ui/js/content.js

```diff
@@ -290,10 +290,10 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
      * @see manageaccess#initManageAccess
```

Comment:
No, we decided that editors can add both other editors and viewers, so they are able to invite other people to work on the document. `manager` should indeed be the default for groups and users added during creation.